.github/workflows/ci.yaml

```diff
@@ -81,4 +95,39 @@ jobs:
       with:
         name: nvidiadriver-e2e-test-logs
         path: ./logs/
-        retention-days: 15  
+        retention-days: 15
+
+    - name: Precompiled e2e test- upgrade kernel and Validate gpu driver
```

Comment:
This is not the right place to define this job. Let's discuss this. But I think this belongs in the scheduled workflow we have for precompiled driver containers that is defined here: https://github.com/NVIDIA/gpu-driver-container/blob/main/.github/workflows/precompiled.yaml

Reply:
I created a new .github/workflows/ci-precompiled.yaml file that can be ported or appended to the pre-compiled scheduled ci file